Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConcepTypeDeleteController.java

```diff
@@ -63,7 +64,8 @@ public String prepareDeleteType(@PathVariable("typeid") String typeid, ModelMap
         boolean enableDelete = true;
         List<ConceptList> conceptLists = conceptListManager.getAllConceptLists();
         for (ConceptList conceptList : conceptLists) {
-            List<ConceptEntry> conceptEntries = conceptManager.getConceptListEntries(conceptList.getConceptListName());
+            List<ConceptEntry> conceptEntries = conceptManager.getConceptListEntries(conceptList.getConceptListName(),
+                    1, -1, "id", IConceptDBManager.DESCENDING);
```

Comment:
if i'm not mistaken,this will break the delete. It's supposed to check all concept for the given type, but you retrieve only the first page, don't you? 

Checking all concept should be done differently (asking directly the database for concept with a specific type) independently of your code, but I think the way it is right will break it.